src/main/java/bio/terra/cli/businessobject/Workspace.java

```diff
@@ -250,13 +253,15 @@ public List<Resource> listResourcesAndSync() {
   /**
    * Clone the current workspace into a new one
    *
+   * @param userFacingId - user-facing ID of the new workspace
    * @param name - name of the new workspace
    * @param description - description of the new workspace
    * @return - ClonedWorkspace structure with details on each resource
    */
-  public ClonedWorkspace clone(@Nullable String name, @Nullable String description) {
+  public ClonedWorkspace clone(
+      @Nullable String userFacingId, @Nullable String name, @Nullable String description) {
```

Comment:
Please change `@Nullable String userFacingId` to `String userFacingId`. When you run `terra workspace clone`, `--new-id` is required. `@Nullable` is only for optional fields. The variable will be null if user didn't pass in that option.